mempool/clist_mempool.go

```diff
@@ -959,12 +1049,11 @@ func (iter *CListIterator) PickLane() types.Lane {
 			continue
 		}
 
-		if iter.counters[lane] >= uint32(lane) {
+		if iter.counters[lane] >= uint(lane) {
```

Comment:
Isn't this logic overlapping with what is now implemented in `NonBlockingWRRIterator) Next()` ? Could we further simplify if here?

Reply:
Probably it's overlapping. But PickLane is not in its final state. Once that code is stable we can see if we can refactor.